Project/app/src/main/java/org/ionproject/android/common/ionwebapi/IonService.kt

```diff
@@ -1,11 +1,13 @@
 package org.ionproject.android.common.ionwebapi
 
 import retrofit2.http.GET
+import retrofit2.http.Headers
 import retrofit2.http.Url
 
-interface IonService {
+private const val AUTHORIZATION_TOKEN = "5tuMrTm8dD-n2d53e3z2ALYC5JRUshAhZUa7xp3cHH4"
```

Comment:
We cannot forget this here. Have you looked at solutions? For example, a properties file that is excluded from the repo.